### PR TITLE
Add two VU9P cards (FX609/S241)

### PR DESCRIPTION
**Pull request by luyong6**
Following changes were squashed.  So just focus on the files that have been changed.

### CODE REVIEW

File: hardware/setup/snap_impl_step.tcl [hunk 1]
```diff
@@ -53,7 +53,11 @@ if { $impl_flow == "CLOUD_BASE" } {
   set opt_route_directive [get_property STEPS.POST_ROUTE_PHYS_OPT_DESIGN.ARGS.DIRECTIVE [get_runs impl_1]]
 }
 
-
+## Temp
+if { ($fpgacard == "FX609") || ($fpgacard == "S241") } {
+  read_xdc $root_dir/setup/$fpgacard/capi_bsp_io.xdc >> $logfile
```
[the remark this answers is not in the document]
Comment (luyong6) [reply hardware/setup/snap_impl_step.tcl]: Hi this is the annoying part. 
In the beginning, I don't have them at this place. But these two xdc files didn't take effect.  My setting for PCIe reference clock pin package didn't take effect and it cost me nearly two weeks to understand why the card didn't work. 
So please put it here for now. And when I upgrade to 2018.2 vivado I may remove them and try again.

**Comment by jsvogt** [comment]
The DCO check hit on the first Signed-off-by message. Both commits in this pull request have valid signed-off messages, so we can proceed here.
:warning: In general it's not a good idea to squash/merge commits from (several) other people, as the person squashing the commits essentially takes responsibility for all commits. It's not clear who edited which line.

**Comment by luyong6** [comment]
I completely agree. The side work had taken too long. It started in April. I have educated my team to sign everything when they commit. And I think with the code going to stabilize, next time it will not bring so many mixed commits for a new card.

**Comment by luyong6** [comment]
I tested one bitstream and it worked well on FX609.

**Comment by luyong6** [comment]
Some fails were due to lack of Vivado licenses. Will they get reran? @jsvogt

**Comment by boekholt** [comment]
Performing "Squash and merge" instead of "Rebase and merge" due to conflicts.

**Pull request merged**